Samples/XboxWdpDriver/Operations/AppOperation.cs

```diff
@@ -70,6 +76,25 @@ public static void HandleOperation(DevicePortal portal, ParameterHelper paramete
                     packagesTask.Wait();
                     Console.WriteLine(packagesTask.Result);
                 }
+                else if (operationType.Equals("listrunningapps") || operationType.Equals("listsuspendedapps"))
+                {
+                    string appState = operationType.Equals("listrunningapps") ? "Running" : "Suspended";
+                    Task<List<string>> packagesTask = portal.GetAppListAsync(appState);
+
+                    packagesTask.Wait();
+                    if (packagesTask.Result.Count > 0)
+                    {
+                        Console.WriteLine(appState + " Apps:");
+                        for (int i = 0; i < packagesTask.Result.Count; i++)
```

Comment:
nit:  For future reference, C# has a nice syntax for loops to omit indexers, something like:
```
foreach (var result in packagesTask.Result)
{
    Console.WriteLine(result);
}
```